app/controllers/api/teams.php

```diff
@@ -241,6 +271,51 @@
         $response->dynamic($team, Response::MODEL_TEAM);
     });
 
+App::patch('/v1/teams/:teamId/prefs')
```

Comment:
Actually PUT should be used here, but PATCH is actually the right way Method for name - since it only changes the name of the team resource - not all.